lib/jsdom/browser/Window.js

```diff
@@ -149,6 +151,13 @@ function Window(options) {
   this.dispatchEvent = this.dispatchEvent.bind(this);
 
   ///// METHODS
+  this.update = function (privateData) {
```

Comment:
You can't add a new nonstandard method to `window`. Instead, move the contents of this to lib/jsdom.js, which is allowed to access `window._document`.